synology-letsencrypt.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash -e
 
 [[ $EUID == 0 ]] || { echo >&2 "This script must be run as root"; exit 1; }
+if [[ "${1:-xxx}" == "xxx" ]]; then
```

Comment:
Introducing arguments is fine, but to future-proof it a little, I'd like it to use getopt and have help messages -- see `install.sh` for a small example.